edx-platform/utec/lms/templates/footer.html

```diff
@@ -62,3 +72,66 @@
   % endfor
 % endif
 
+%if modal_force_display:
+## Pop-up tos
+<div class="modal-container">
+  <div class="modal-content">
+    % if modal_tos_text:
```

Comment:
Is this necessary? if you have a default value, then `modal_tos_text` will never be empty, or none